pyhf/__init__.py

```diff
@@ -336,17 +361,17 @@ def runOnePoint(muTest, data, pdf, init_pars, par_bounds):
     qmuA_v = qmu(muTest, asimov_data, pdf, init_pars, par_bounds)
 
     try:
-        sqrtqmu_v = math.sqrt(qmu_v)
+        sqrtqmu_v = tensorlib.sqrt(qmu_v)
     except ValueError:
         log.warning('WARNING: qmu_v negative: %s', qmu_v)
         qmu_v = tensorlib.tolist(tensorlib.clip(qmu_v, 0, max=None))
-        sqrtqmu_v = math.sqrt(qmu_v)
+        sqrtqmu_v = tensorlib.sqrt(qmu_v)
     try:
-        sqrtqmuA_v = math.sqrt(qmuA_v)
+        sqrtqmuA_v = tensorlib.sqrt(qmuA_v)
     except ValueError:
         log.warning('WARNING: qmuA_v negative: %s', qmuA_v)
         qmuA_v = tensorlib.tolist(tensorlib.clip(qmuA_v, 0, max=None))
-        sqrtqmuA_v = math.sqrt(qmuA_v)
+        sqrtqmuA_v = tensorlib.sqrt(qmuA_v)
 
     sigma = muTest / sqrtqmuA_v if sqrtqmuA_v > 0 else None
 
```

Comment:
actually I think we can just remove this, we don't actually use this anywhere, I thin it's just a relic from some tests I did

Reply:
Ah, okay. I'll remove `sigma` everywhere in `pvals_from_teststat` then

Reply:
I think it's just in the line where it's defined and the return line

Reply:
Yeah, that's right. The tests will of course still fail after 9ea1264 as I haven't tensorized `pvals_from_teststat` yet.